rabbitmq/hatch.toml

```diff
@@ -9,5 +9,9 @@ dependencies = [
 python = ["27", "38"]
 version = ["3.5", "3.6", "3.7"]
 
+[[envs.default.matrix]]
+python = ["38"]
+version = ["3.8", "3.9", "3.10", "3.11"]
+
```

Comment:
Shouldn't this be merged with or replace the entry right above?

Reply:
This is the only way I knew of making sure rmq 3.8+ is *only tested with python 3*. Perhaps there's some hatch-foo that I'm missing? 🤷

Reply:
No, it was actually me being unfamiliar with hatch's syntax for defining matrices 🤦 .